doc/api/cli.md

```diff
@@ -1577,6 +1577,32 @@ on the number of online processors.
 If the value provided is larger than V8's maximum, then the largest value
 will be chosen.
 
+### `--watch`
+
+<!-- YAML
+added: REPLACEME
+-->
+
+> Stability: 1 - Experimental
+
+Starts the Node.js in watch mode.
+When in watch mode, changes in the watched files cause node to restart.
+By default, watch mode will watch the entry point
+and any required module. use `--watch-path` to specify what paths to watch
+
+This flag cannot be combined with
+`--check`, `--eval`, `--interactive`, or the REPL.
+
+### `--watch-path`
+
+<!-- YAML
+added: REPLACEME
+-->
+
+> Stability: 1 - Experimental
+
+Specify what paths to watch in watch mode.
```

Comment:
Clarify (doc wise) if this is _in addition_ to the import/require graph or instead of + an example?